### PR TITLE
designed ratings, toast and grids

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

**Review by Omkar-Ghate — APPROVED** [review]
Looks good to merge.

File: Components/Grid/Grid.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.grid-autofit-breaks > * {
+  border: 0.5px solid hsl(var(--clr-dark));
+}
+
+@media (min-width: 0) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(2rem, 1fr));
+  }
+}
```
Comment (zoef-yt) [inline Components/Grid/Grid.css:48]: Add spacing

File: Components/Grid/Grid.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+@media (min-width: 60em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));
+  }
+}
+@media (min-width: 75em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+  }
+}
```
Comment (zoef-yt) [inline Components/Grid/Grid.css:68]: Add spacing

File: Components/Grid/Grid.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+@media (min-width: 45em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+  }
+}
+@media (min-width: 60em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));
+  }
+}
```
Comment (zoef-yt) [inline Components/Grid/Grid.css:63]: Add spacing

File: Components/Grid/Grid.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+@media (min-width: 30em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(5rem, 1fr));
+  }
+}
+@media (min-width: 45em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+  }
+}
```
Comment (zoef-yt) [inline Components/Grid/Grid.css:58]: Add spacing

File: Components/Grid/Grid.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+@media (min-width: 0) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(2rem, 1fr));
+  }
+}
+@media (min-width: 30em) {
+  .grid-autofit-breaks {
+    grid-template-columns: repeat(auto-fit, minmax(5rem, 1fr));
+  }
+}
```
Comment (zoef-yt) [inline Components/Grid/Grid.css:53]: Add spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by srishtigautam21 — APPROVED** [review]
seems fine to me...good to merge

**Review by tonymike99 — APPROVED** [review]
Great code. Good use of autofit and minmax!

File: Components/Grid/Grid.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/Grid/Grid.css:48]: noted

File: Components/Grid/Grid.css (hunk 2 above)
Comment (itsmebipul00) [reply Components/Grid/Grid.css:68]: noted

File: Components/Grid/Grid.css (hunk 3 above)
Comment (itsmebipul00) [reply Components/Grid/Grid.css:63]: noted

File: Components/Grid/Grid.css (hunk 4 above)
Comment (itsmebipul00) [reply Components/Grid/Grid.css:58]: noted

File: Components/Grid/Grid.css (hunk 5 above)
Comment (itsmebipul00) [reply Components/Grid/Grid.css:53]: noted

**Pull request merged**